### PR TITLE
add logging for banmanager

### PR DESCRIPTION
**Pull request by jjj5311**
initial fix for #33 i am unfamiliar with maven so i kind of botched it with the .classpath

however the rest is doing logging like asked... i cant test anymore since i am not banned

### CODE REVIEW

File: src/com/untamedears/PrisonPearl/CombatTagListener.java [hunk 1]
```diff
@@ -25,7 +27,7 @@ public void onNpcDespawn(NpcDespawnEvent event) {
         if (event.getReason() != NpcDespawnReason.DESPAWN_TIMEOUT) {
             return;
         }
-        UUID plruuid = event.getPlayerUUID();
+        UUID plruuid = NameAPI.getUUID(event.getPlayerName());
```
Comment (rourke750) [inline src/com/untamedears/PrisonPearl/CombatTagListener.java]: PP has a soft dependency on namelayer so you need to check if namelayer is installed first.  Look at some of the other places where nameAPi checks are called.

Comment (jjj5311) [inline src/com/untamedears/PrisonPearl/CombatTagListener.java]: That change isn't necessary if it was working the other way, I was getting
errors though and couldn't compile to test

Comment (rourke750) [inline src/com/untamedears/PrisonPearl/CombatTagListener.java]: Ah I see what you mean.  Are you running the lastest combat tag?  getPlayerUUID is in the most current source https://github.com/cheddar262/Combat-Tag/blob/master/src/main/java/com/trc202/CombatTagEvents/NpcDespawnEvent.java

Comment (jjj5311) [inline src/com/untamedears/PrisonPearl/CombatTagListener.java]: Probably not then I was having problems getting that to compile so I just
pulled one off Jenkins

Comment (rourke750) [inline src/com/untamedears/PrisonPearl/CombatTagListener.java]: I should take down the combat tag off our jenkins page as we no longer depend on our custom implementation.

Comment (jjj5311) [inline src/com/untamedears/PrisonPearl/CombatTagListener.java]: good idea,

want me to amend the commit?

Comment (rourke750) [inline src/com/untamedears/PrisonPearl/CombatTagListener.java]: yes pls.

**Comment by jjj5311** [comment]
updated

**Comment by rourke750** [comment]
@erocs Is this a good format for you?

**Comment by rourke750** [comment]
@jjj5311 Just take erocs when ever you want him to look again.

**Pull request merged**